examples/ABMC.yml

```diff
@@ -0,0 +1,14 @@
+abbreviation: ABMC
+departments:
+- name: American Battle Monuments Commission
+  submission_methods:
+  - submission_format: usps
+    name: Beatriz Collazo
+    title: FOIA Assistant
```

Comment:
You mean including the `name` or `title` as part of `address_lines`? Maybe an example of the alternative would help clarify.